# Request
---
bool connected
string ssid
